RHEcosystemAppEng/dbaas-operator
# use the environment variable from operator subscription

## Description


- > update the code to add params for observatorium into operator subscription 
- > update the code to read the observatorium token from secret.
- > add installation file steps for non OSD cluster. 

## Verification Steps


1. Verify the params are part of operator subscription.
2. Verify by creating secret, and verify the remote is working accordingly.

## Type of change


## Checklist (Definition of Done)
